plugins/ModuleLevelTrigger.cpp

```diff
@@ -330,6 +344,16 @@ ModuleLevelTrigger::send_trigger_decisions()
   while (m_running_flag) {
     std::optional<triggeralgs::TriggerCandidate> tc = m_candidate_input->try_receive(std::chrono::milliseconds(10));
     if (tc.has_value()) {
+
+      if (m_first_tc) {
+        m_initial_offset = (duration_cast<milliseconds>(system_clock::now().time_since_epoch()).count()) - (tc->time_start*(16*1e-6));
+        m_first_tc = false;
+      }
```

Comment:
Just in case, was this tested with the replay and with some real detector run (I guess warmbox)?

Reply:
tested with the default file from instructions page and with replay, not with real detector yet (can only be tested once in patch, next week)